packages/office-ui-fabric-react/src/components/Image/Image.base.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import { BaseComponent, classNamesFunction, getNativeProps, imageProperties, createRef } from '../../Utilities';
-import { IImageProps, IImageStyles, IImageStyleProps, ImageCoverStyle, ImageFit, ImageLoadState } from './Image.types';
+import { BaseComponent, classNamesFunction, createRef, getNativeProps, imageProperties } from '../../Utilities';
```

Comment:
You can use `React.createRef` rather than Utilities `createRef` which was only there when the React version wasn't available.